services/web/server/tests/unit/with_dbs/_helpers.py

```diff
@@ -0,0 +1,71 @@
+from asyncio import Future
+from collections import namedtuple
+from typing import List, Tuple
+
+from aiohttp import web
+
+from simcore_service_webserver.projects.projects_handlers import HTTPLocked
+from simcore_service_webserver.security_roles import UserRole
+
+ExpectedResponse = namedtuple(
+    "ExpectedResponse",
+    ["ok", "created", "no_content", "not_found", "forbidden", "locked"],
+)
+
+
+def standard_role_response() -> Tuple[str, List[Tuple[UserRole, ExpectedResponse]]]:
+    return (
```

Comment:
BTW: how did you make it work??

Reply:
some bloody space/parenthis or so I guess...